Scripts/bootstrap.sh

```diff
@@ -43,13 +35,20 @@ ln -sf ../../hooks/pre-commit.sh .git/hooks/pre-commit
 chmod +x .git/hooks/pre-commit
 chmod +x ./hooks/git-format-staged
 
-# Install gems
 puts "Install bundle dependencies"
 bundle install
 
-# Copy internal Xcode scheme to the right folder for
-puts "Adding DemoApp-StreamDevelopers.xcscheme to the Xcode project"
-cp Scripts/DemoApp-StreamDevelopers.xcscheme StreamChat.xcodeproj/xcshareddata/xcschemes/DemoApp-StreamDevelopers.xcscheme
+if [[ ${XCODE_ACTIONS-default} == default ]]; then
+  puts "Install Mint if needed"
+  brew install mint
+
+  puts "Bootstrap Mint dependencies"
+  mint bootstrap
+
+  # Copy internal Xcode scheme to the right folder for
+  puts "Adding DemoApp-StreamDevelopers.xcscheme to the Xcode project"
+  cp Scripts/DemoApp-StreamDevelopers.xcscheme StreamChat.xcodeproj/xcshareddata/xcschemes/DemoApp-StreamDevelopers.xcscheme
+fi
```

Comment:
What are the cases when we bypass installing mint? When `XCODE_ACTIONS-default != default`?

Reply:
At the moment in the `allure_testops` job. Might be something else as well in the future. By default `${XCODE_ACTIONS-default}` always equals to `default`.